plugins/tiddlywiki/multiwikiserver/modules/routes/handlers/post-bag.js

```diff
@@ -26,6 +28,7 @@ exports.bodyFormat = "www-form-urlencoded";
 exports.csrfDisable = true;
 
 exports.handler = function(request,response,state) {
+	aclMiddleware(request, response, state, "bag", "WRITE");
```

Comment:
It looks like the middleware call can fail authentication, and send a response to the client. Presumably we would need some processing here to respond to the outcome of the middleware.

Is it necessary for each route to invoke the middleware explicitly like this? I was hoping the main handler in `mws-server.js` could handle the middleware more or less invisibly from the point of view of the routes.